includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -189,7 +189,7 @@ public function __construct(
 					'<a href="https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/" target="_blank">Apple</a>'
 				),
 				'type'        => 'checkbox',
-				'description' => __( 'If enabled, users will be able to pay using Apple Pay or Chrome Payment Request if supported by the browser.', 'woocommerce-payments' ),
+				'description' => __( 'If enabled, users will be able to pay using Apple Pay, Google Pay or the Payment Request API if supported by the browser.', 'woocommerce-payments' ),
```

Comment:
@ricardo are this changes visible in the UI?

Reply:
Yes. This is the tooltip text under **Payments > Settings > Payment request buttons > Enable/disable**. I removed Chrome Payment Request because it doesn't work only in Chrome, but in other supported browsers, and added Google Pay.

![image](https://user-images.githubusercontent.com/5509901/118579051-b1f1ef00-b763-11eb-9368-f868a0432ab4.png)